app/assets/stylesheets/nav.scss

```diff
@@ -116,7 +124,11 @@ body.home {
 }
 
 .sub-nav {
-  margin-left: 18%;
+  dd a {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Selector should have depth of applicability no greater than 2, but was 3